topiary-cli/src/error.rs

```diff
@@ -134,3 +147,12 @@ impl From<serde_toml_merge::Error> for TopiaryError {
         )
     }
 }
+
+impl From<tokio::task::JoinError> for TopiaryError {
+    fn from(e: tokio::task::JoinError) -> Self {
+        TopiaryError::Bin(
+            format!("Could not join parallel formatting tasks: {}", e),
```

Comment:
You shouldn't have to display the inner message from `e`. Because you supply `e` as an inner error (source), it is the error reporter's responsibility to show the inner message.